maestro-client/src/main/java/maestro/drivers/IOSDriver.kt

```diff
@@ -515,6 +515,16 @@ class IOSDriver(
         LOGGER.warn("Airplane mode is not available on iOS simulators")
     }
 
+    override fun isDarkModeEnabled(): Boolean {
+        val deviceId = iosDevice.deviceId ?: "booted"
+        return XCRunnerCLIUtils.isDarkModeEnabled(deviceId)
```

Comment:
Is there a way to do this via xcuitest? We're trying to move away from simctl commands since they won't work for physical devices

Reply:
I'll look into this tomorrow to see if there's a way to do it. Not sure at the moment.

Reply:
@Leland-Takamine I found this reference already inside the repo's source code, but i have no idea how i would go about using it 🤷  https://github.com/mobile-dev-inc/Maestro/blob/main/maestro-ios-xctest-runner/maestro-driver-iosUITests/PrivateHeaders/XCTest/XCUIDevice.h#L18-L22